README.md

```diff
@@ -1 +1,30 @@
-# sc18-poster
\ No newline at end of file
+# SC-18 poster
+## Copyright (c) 2018 Caporaso Lab
+Details in LICENSE.md
+
+## Gemini theme copyright (c) 2018 Anish Athalye (me@anishathalye.com)
+License details inline
+
+Readme text below modified from: [Gemini README] and subject to [Gemini License]
+
+## Dependencies
+
+* A TeX installation that includes [LuaTeX]
+* LaTeX package dependencies including beamerposter (these usually come with
+  your TeX installation, but if not, you can get them from [CTAN])
+* [Raleway] and [Lato], which are both available under Open Font License
+
+## Usage
+
+1. Copy the files in this repository (or clone the repository)
+
+1. Run `make` to build the poster
+
```

Comment:
This looks great @ChrisKeefe! Do you think we should add a line for `make clean` in case the user wants to redo their poster?